airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/write/InputConsumer.kt

```diff
@@ -68,6 +70,7 @@ class DefaultInputConsumer(
 @Factory
 class InputStreamFactory {
     @Singleton
+    @Requires(notEnv = [Environment.TEST])
```

Comment:
If we don't add this, the bean defined by the `CliRunner` will collide with this factory.